backend/app/Http/Controllers/API/DisplayController.php

```
@@ -63,6 +64,30 @@ public function index(): JsonResponse
+        if (! $permission->permitted) {
+            return $this->error(message: $permission->message, code: $permission->code);
+        }
+
+        try {
+            $date = $request->query('date')
+                ? Carbon::parse($request->query('date'))->startOfDay()
+                : null;
+
+            $events = $this->eventService->getEventsForDisplay($displayId, $date);
+
+            return $this->success(data: EventResource::collection($events));
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Http\Resources\API\EventResource::collection().


(staticMethod.notFound)